docs/source/development/eep-00-governance-model.md

```diff
@@ -0,0 +1,125 @@
+```
+
+## Purpose
+
+This document formalizes the estimagic code of conduct and governance model. In case
+of changes, this document can be updated following the Estimagic Enhancement Proposal
+process detailed below.
+
+
+
+```{include} ../../../CODE_OF_CONDUCT.md
+```
+
+## Estimagic governance model
+
+### Summary
+
+The governance model strives to be lightweight and based on [consensus](https://numpy.org/doc/stable/dev/governance/governance.html#consensus-based-decision-making-by-the-community) of all interested parties. Most work happens in GitHub issues and pull requests (regular
+decision process). Any interested party can voice their concerns or veto on proposed
+changes. If this happens, the Estimagic Enhancement Proposal (EEP) process can
+be used to iterate over proposals until consesus is reached (controversial
+decision process). If necessary, members of the steering council can moderate heated
+debates and help to broker a consensus.
+
+### Regular decision process
+
+Most changes to estimagic are additions of new functionality or strict improvements
+of existing functionality. Such changes can be discussed in GitHub issues and
+discussions and implemented in pull requests. They do not require an Estimagic
+Enhancement Proposal.
+
+Before starting to work on estimagic, contributors should read [how to contribute](how-to)
+and the [styleguide](styleguide). They can also reach out to existing contributors if
+any help is needed or anything remains unclear. We are all happy to help onboarding new
+contributors in any way necessary. For example, we have given introductions to git and
+GitHub in the past to help people make a contribution to estimagic.
+
+Pull requests should be opened as soon as work is started. They should contain a good
+description of the planned work such that any interested party can participate in the
+discussion around the changes. If planned changes turn out to be controversial, their
+design should be discussed in an Estimagic Enhancement Proposal before the actual
+work starts. When the work is finished, the author of a pull
+request can request a review. In most cases, previous discussions will show who is a
+suitable reviewer. If in doubt, tag [janosg](https://github.com/janosg). Pull requests
+can be merged if there is at least one approving review.
+
+Reviewers should be polite, welcoming and helpful to the author of the pull request
+who might have spent many hours working on the changes. Major points should be discussed
+publicly on GitHub, but very critical feedback or small details can be moved to private
+discussions. Video calls can help if a discussion gets stuck. Keep in mind that many
+people get notified on every review comment (see [here](https://rgommers.github.io/2019/06/the-cost-of-an-open-source-contribution/) for an execellent discussion).
+The code of conduct applies to all interactions related to code reviews.
+
+### Estimagic Enhancement Proposals (EEPs) / Controversial decision process
```

Comment:
If the convention is to have "estimagic" always in lowercase, I do not think we should capitalize in titles or as first word of a sentence, either. But no strong opinion. Should just be consistent, e.g. line 80 is lowercase in the middle of some text but line 109 capitalised.